WKWebView.android.js

```diff
@@ -200,6 +200,13 @@ class WebView extends React.Component {
       'always',
       'compatibility'
     ]),
+    /**
+     * A RefreshControl component, used to provide pull-to-refresh
+     * functionality for the WebView.
+     *
+     * See [RefreshControl](docs/refreshcontrol.html).
```

Comment:
docs/refreshcontrol.html ?

Reply:
sorry , it is https://facebook.github.io/react-native/docs/refreshcontrol.html